x-pack/plugins/observability/server/services/slo/get_slo.ts

```diff
@@ -26,6 +26,9 @@ export class GetSLO {
       time_window: slo.time_window,
       budgeting_method: slo.budgeting_method,
       objective: slo.objective,
+      revision: slo.revision,
+      created_at: slo.created_at,
```

Comment:
What has changed that now you return `created_at` and `updated_at` fields?

Reply:
I've thought a bit more about the UI, and used the Alerts page as example, and I think we'll have similar display and sorting functionalities, which includes sorting per creation/update time for example